### ISSUE TITLE
getInitialProps not passed down in NextJS

### ISSUE THREAD

**Issue by alcesleo** [panz3r/react-keycloak#41]
**Describe the bug**
When using `getInitialProps` on a NextPage in an app using `appWithKeycloak`, the page crashes and never gets the props - even if the page itself doesn't use react-keycloak functionality.

**To Reproduce**
1. In the example project for NextJS, create a new page with `getInitialProps`:

```
import { NextPage } from 'next'

interface IProps {
  message: string
}

const BugPage: NextPage<IProps> = ({ message }) => {
  return <p>{message}</p>
}

BugPage.getInitialProps = async () => {
  return { message: 'Hello World!' }
}

export default BugPage
```

2. Try to load the page - it crashes
3. Comment out the `getInitialProps` -> the page loads again
4. Uncomment the `getInitialProps` and comment the use of `appWithKeycloak` in _app.tsx -> the page also loads fine

**Expected behavior**
`getInitialProps` can be used as usual

**Environment**
 - OS: Windows 10
 - Browser: Chrome 79
 - Version: 1.0.0-alpha.9

**Comment by panz3r** [comment]
Hi @alcesleo ,

Thanks for your feedback and sorry for the inconveniences this issue might have caused.

I think I've found the cause of it and I'm going to release a fix as soon as possible.

**Comment by alcesleo** [comment]
My pleasure, thank you for the quick response! Let me know if you need any more information.

**Comment by panz3r** [comment]
Hi @alcesleo ,

Thanks again for your feedback, I was able to reproduce and fix the reported issue.

The fix has been released with version `1.0.0-alpha.10` of `@react-keycloak/nextjs` alongside some major improvements.

**Comment by alcesleo** [comment]
Nice work! It seems to have solved my issue, thanks a lot!
